### DISCUSSION TITLE
flakey question: is this flake appropriate for a bare metal installation

### DISCUSSION THREAD

**Discussion by StInnocent** [drakon64/nixos-cosmic-iso#15] [General]
Is the flake provided ( https://github.com/drakon64/nixos-cosmic-iso/blob/main/flake.nix ) appropriate to build a bare metal installation of NixOS providing Cosmic?

**Comment by drakon64** [comment]
The Flake is for creating a Live ISO, it isn't going to do anything useful for a live system.

If you want to build a live system with COSMIC, I suggest you look at https://github.com/lilyinstarlight/nixos-cosmic.
